mob_test.go

```diff
@@ -166,6 +166,31 @@ func TestNextNotMobProgramming(t *testing.T) {
 	assertOutputContains(t, output, "you aren't mob programming")
 }
 
+func TestRequireCommitMessage(t *testing.T) {
+	output := setup(t)
+
+	os.Unsetenv("MOB_REQUIRE_COMMIT_MESSAGE")
+	defer os.Unsetenv("MOB_REQUIRE_COMMIT_MESSAGE")
+
+	configuration = parseEnvironmentVariables(getDefaultConfiguration())
+	equals(t, false, configuration.RequireCommitMessage)
+
+	os.Setenv("MOB_REQUIRE_COMMIT_MESSAGE", "false")
+	configuration = parseEnvironmentVariables(getDefaultConfiguration())
+	equals(t, false, configuration.RequireCommitMessage)
+
+	os.Setenv("MOB_REQUIRE_COMMIT_MESSAGE", "true")
+	configuration = parseEnvironmentVariables(getDefaultConfiguration())
+	equals(t, true, configuration.RequireCommitMessage)
```

Comment:
For boolean environment variables, there's a parameterized test available: ` TestBooleanEnvironmentVariables`. You can simply add your variable `MOB_REQUIRE_COMMIT_MESSAGE` and it will be covered by four tests. And you can simplify the test above as you just set the `configuration.mobRequireCommitMessage = true` without any environment variables.